docs/fleet/fleet.md

```diff
@@ -0,0 +1,77 @@
+title: Fleet Overview
+description: Overview of Fleet - Sematext Agent management panel
+
+The Fleet section of Fleet and Discovery gives you an overview of all the Sematext Agents that are running in your systems.
+
+![Fleet Screen](../images/fleet/fnd-agents-collapsed.png)
+
+At first glance, you can see the following data in the table:
+
+- Host: Names of the hosts where the Sematext Agent is installed. You can either click on the name to open the Agent Information Panel or click on the expand view to reveal the list of Discovered Services available on the specific host. For further details, refer to the individual sections below in the document.
```

Comment:
Names of the hosts ==> Names of hosts

> where the Sematext Agent is installed

Actually, it's more about where it's running, right?
Or where it was running within the last N hours?
Which makes me think of 2 things:

1. People sometimes ask about what N is.  The answer is that the host/agent will disappear from this list after a while.  We should document that N.  And maybe we can also just say that somewhere in the UI?
2. Installed vs. running.  You know how we have inventory > packages?  Imagine if we compared that to the list of hosts where STA is running.  We could have a screen, maybe under F&D or maybe elsewhere that shows the "diff".  Know what I mean?  Hosts where you have STA running, but it's not running.  We have the data.  Also have STA version data, so can label those that could be updated.  We could also link to docs where we show commands for "parallel STA start" and  "parallel STA update" on N hosts.  See what I mean? Mmmmmmm. Ticket material.  Straight forward UI and impl.